Rakefile

```diff
@@ -14,12 +16,25 @@ end
 
 namespace :design_system do
   desc "All Design System Tests"
-  task :all do
+  task all: [:report_folders] do
     puts "Running all design system tests"
     system(
       "cd ./testing && bundle exec cucumber -p reports && cd .."
```

Comment:
This here, the first step. May be needed also in the report folders test. Or alternatively you can add the testing/ prefix

Reply:
Added the testing prefix.